WordPress/src/main/res/layout/my_site_info_card.xml

```diff
@@ -4,17 +4,17 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingBottom="@dimen/margin_medium"
     android:paddingEnd="@dimen/margin_medium_large"
-    android:paddingStart="@dimen/content_margin_site_row_start"
-    android:paddingTop="@dimen/margin_extra_large"
     tools:ignore="RtlSymmetry">
 
     <com.google.android.material.card.MaterialCardView
```

Comment:
Suggestion (💡): Apologies, it was my fault to have to change that and now you rolled it back to the previous version since it is necessary for the quick start blue dot to appear as expected. Having said that, and in order to avoid something similar happening in the future with a new couple of engineering doing the same, what if we add a comment here letting any future engineer know why this is needed, quick start blue dot and all. Wdyt? 🤔

Reply:
No need to apologize, please. 🙏 We were only trying to make the margins consistent across the cards, and it was a very good suggestion.

I've added a note in the XML in c7df015.

Reply:
👋 @ashiagr !

> No need to apologize, please. 🙏 We were only trying to make the margins consistent across the cards, and it was a very good suggestion.

🥇 

> I've added a note in the XML in c7df015.

Looking good, many thanks!